deploy/main.tf

```diff
@@ -7,7 +7,6 @@
 ######################################
 terraform {
   required_version = ">= 1.0.1"
-  backend "remote" {}
```

Comment:
The deploy folder should have default back set for remote. Rest looks good to me.

Reply:
sorted